### PR TITLE
Display languages localized

### LINKED ISSUE #291: Language names should be displayed in their language instead of in English
Instead of showing German, English, French, I think it would make more sense to show

Deutsch, English, Français (?) so that people recognize their language.

Also, finding the language selection menu can get tedious when you selected a different language accidentally.

### PR DESCRIPTION
**Pull request by tobbi**
This allows to display the language names in the options menu in localized form. Also includes some previously missing font files.

Fixes #291

### CODE REVIEW

File: src/gui/menu.hpp [hunk 1]
```diff
@@ -78,7 +79,7 @@ class Menu
 
   ItemHorizontalLine& add_hl();
   ItemLabel& add_label(const std::string& text);
-  ItemAction& add_entry(int id, const std::string& text);
+  ItemAction& add_entry(int id, const std::string& text, FontPtr font = Resources::normal_font);
```
Comment (Vankata453) [inline src/gui/menu.hpp:82]: There shouldn't be a need to provide the font in the `Menu` constructor function. Could be provided on the returned `ItemAction`.

**Review by Vankata453 — CHANGES_REQUESTED** [review]

**Comment by Vankata453** [comment]
image

Noticing these two issues with the language menu, everything else looks fine.

**Comment by tobbi** [comment]
Did you check out the submodule changes as well?

**Comment by Vankata453** [comment]
@tobbi That fixed it, I forgot.

**Review by Vankata453 — APPROVED** [review]

**Pull request merged**